react-app/package.json

```
@@ -12,7 +12,11 @@
     "check": "prettier --check \"./src/**/*.{js,jsx,ts,tsx,css,scss}\""
   },
   "dependencies": {
+    "@bcgov/bc-sans": "^2.1.0",
     "@bcgov/citz-imb-kc-react": "https://github.com/bcgov/citz-imb-kc-react/releases/download/v1.1.0/bcgov-citz-imb-kc-react-1.1.0.tgz",
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.15.3",
```

Recommend we lock down these versions.
```suggestion
    "@bcgov/bc-sans": "2.1.0",
    "@bcgov/citz-imb-kc-react": "https://github.com/bcgov/citz-imb-kc-react/releases/download/v1.1.0/bcgov-citz-imb-kc-react-1.1.0.tgz",
    "@emotion/react": "11.11.3",
    "@emotion/styled": "11.11.0",
    "@mui/material": "5.15.3",
```